pip/requirements.in

```diff
@@ -2,7 +2,7 @@
 -e git://github.com/theatlantic/django-news-sitemaps.git@v1.3.0#egg=django_news_sitemaps # For generating news sitemaps
 beautifulsoup4 # used for parsing html
 bleach # Used to sanitize any HTML we're rendering
-boto # Used to access AWS
+boto3 # Used to access AWS
```

Comment:
I regenerated `pip/requirements.txt` after isolating this change, which is why that file has a lot of diffs